### PR TITLE
Post release 0.17.14

### PR DESCRIPTION
**Pull request by Gvaihir**

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 
 ...
 
+## 0.17.14 (2022-10-20)
+
+* Added support for Directory type outputs for WDL 2.0
```
Comment (commandlinegirl) [inline RELEASE_NOTES.md]: How about saying it's the `development` version (to match how we refer to this version in the Readme in dxCompiler as a version that we partially support)

**Review by commandlinegirl — CHANGES_REQUESTED** [review]

File: RELEASE_NOTES.md (hunk 1 above)
Comment (commandlinegirl) [reply RELEASE_NOTES.md]: Here and also in `dxCompiler` release note please. Thanks!

**Review by commandlinegirl — DISMISSED** [review]
But feel free to merge if you're pressed for time, my previous comment is not a blocker.

File: RELEASE_NOTES.md (hunk 1 above)
Comment (Gvaihir) [reply RELEASE_NOTES.md]: fixed in [72743b6](https://github.com/dnanexus/wdlTools/pull/236/commits/72743b623346b3b7534c9fbfe2455d23f7bf4325)

**Review by commandlinegirl — APPROVED** [review]

**Pull request merged**